var/spack/repos/builtin/packages/rocprofiler-dev/package.py

```diff
@@ -126,6 +134,13 @@ class RocprofilerDev(CMakePackage):
         "5.3.3",
         "5.4.0",
         "5.4.3",
+        "5.5.0",
```

Comment:
Did you mean to add `"5.4.4"` here too?

Reply:
Removed 5.4.4 as there is no 5.4.4 releases for "hsakmt-roct and hsa-rocr-dev.